### PR TITLE
344 ux GitHub auth

### PR DESCRIPTION
**Pull request by ottodevs**
This PR completely addresses #344 solution by adding new Github authentication flow and options on Projects App.
Additionally some proptypes were added on related components as well as some typos correction and linting.

GitHub is written with uppercase H, we were previously using Github, so probably we should be changing the occurrences to the correct form.

### CODE REVIEW

**Comment by Quazia** [comment]
Can you resolve the CI issues please?

File: apps/projects/app/components/Content/Issues.js [hunk 1]
```diff
@@ -241,8 +254,7 @@ class Issues extends React.PureComponent {
   }
 
   shapeIssues = issues => {
-    const { tokens, bountyIssues, } =  this.props
-    let bountyIssueObj = {}
```
[the remark this answers is not in the document]
Comment (ottodevs) [reply apps/projects/app/components/Content/Issues.js]: Good catch! Seems like a regression on any of the several dev merges I don't recall modifying anything related to this function.
The problem is sometimes a line is automatically modified by eslint as it happens at L257 and then with subsequent dev branch merges git does not interpret how the merge should be correctly, I usually review every merge line by line, but I missed this one, thanks for pointing out !

**Comment by ottodevs** [comment]
> These changes look really good! I've found several UX/UI issues that may or may not be easily fixed in this enchancement.

I solved these note related to the changes this PR initially intended to implement:
> * [x]  Add some padding aove to the Github decoupling note on the login screen

The rest of changes you ask for seemed complex enough and unrelated to this PR so I captured them here: https://github.com/AutarkLabs/planning-suite/issues/476

File: apps/projects/app/components/Content/Issues.js (hunk 1 above)
[the remark this answers is not in the document]
Comment (topocount) [reply apps/projects/app/components/Content/Issues.js]: Is it possible it's due to the force-pushes?

[the remark this answers is not in the document]
Comment (topocount) [reply apps/projects/app/components/Content/Issues.js]: It's not that important but it looks like it was just deleted in this commit: https://github.com/AutarkLabs/planning-suite/pull/455/commits/191e78fca3e551e2b039ab4e35d0c1f4ece56afe

**Pull request merged**